.mage/magefile.go

```diff
@@ -25,15 +27,36 @@ import (
 	"go.einride.tech/mage-tools/targets/mggoreview"
 )
 
+func init() {
+	mgmake.AddExtraMakefiles(
+		[]mgmake.Makefile{
+			{
+				Path:          mgtool.GetGitRootPath("proto/Makefile"),
+				Namespace:     Proto{},
+				DefaultTarget: "Hej",
```

Comment:
is `Hej` correct here or just a test? :D

Reply:
Yeah, this was mainly to showcase the PR. Will remove before we merge